examples/manipulating_visualizing/plot_visualize_megatrawls_netmats.py

```diff
@@ -0,0 +1,44 @@
+"""
+Visualizing Megatrawls Network Matrices from Human Connectome Project
+=====================================================================
+
+This example shows how to visualize network matrices fetched from
+HCP beta-release of the Functional Connectivity Megatrawl
+
+See :func:`nilearn.datasets.fetch_megatrawls_netmats` documentation for more details.
+"""
+import matplotlib.pyplot as plt
+import numpy as np
+
+from nilearn import datasets
+from nilearn import plotting
+
+
+def plot_mats(netmats, title):
```

Comment:
That's plotting only one matrix, right? If yes, just a naming suggestion: `plot_matrix(matrix, title)`